dubbo-plugin/dubbo-qos/src/main/java/org/apache/dubbo/qos/command/impl/Ls.java

```diff
@@ -58,7 +57,7 @@ public String listProvider() {
 
         //Content
         for (ProviderModel providerModel : ProviderModelList) {
-            tTable.addRow(providerModel.getServiceName(), isReg(providerModel.getServiceName()) ? "Y" : "N");
+            tTable.addRow(providerModel.getServiceName(), isRegistered(providerModel.getServiceName()) ? "Y" : "N");
```

Comment:
Better to define a constant for "Y" and "N".

Reply:
let's leave it as is.